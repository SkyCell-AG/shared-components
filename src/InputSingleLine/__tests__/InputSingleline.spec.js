import React from 'react'
import {
    shallow,
    mount,
} from 'enzyme'

import InputSingleline from '../InputSingleLineContainer'

describe('shared-components/InputSingleline', () => {
    it('should match snapshot', () => {
        const wrapper = shallow(
            <InputSingleline
                value="Text 234"
                onChange={jest.fn()}
            />,
        )

        expect(wrapper).toMatchSnapshot()
    })

    it('mount correct without data', () => {
        mount(
            <InputSingleline
                onChange={jest.fn()}
            />,
        )
    })
})
