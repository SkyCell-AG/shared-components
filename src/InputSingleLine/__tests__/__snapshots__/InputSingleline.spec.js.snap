// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared-components/InputSingleline should match snapshot 1`] = `
<InputSingleLine
  className=""
  classNames={Object {}}
  disabled={false}
  fullHeight={false}
  fullWidth={false}
  hasError=""
  hideUnderline={false}
  onBlur={[Function]}
  onChange={[MockFunction]}
  onKeyPress={[Function]}
  required={false}
  rightAligned={false}
  suffix=""
  title=""
  type="text"
  value="Text 234"
/>
`;
